### ISSUE TITLE
Very low framerate

### ISSUE THREAD

**Issue by AlexandreBonneau** [virtual-programming/dirtshowdown-linux#36]
Hey, until the last time I played, Dirt showdown was always very responsive, with a 2560x1440 resolution, 2xMSAA, most settings to ultra, etc., on my Debian Testing with a Radeon HD7950, mesa 13.0.2, a core i5 2500k and 16Gio of RAM.
I'm also using KDE with kwin_x11 4:5.8.2-1+b1, xserver-xorg-video-amdgpu 1.2.0-1+b1 and libdrm-amdgpu1:amd64 2.4.74-1.

Today I launch the game and the resolution was 720p. When I tried to change it back to 2560x1440, the framerate dropped to a mere 0.5fps perhaps, making it near impossible to even navigate the menu.

What changed lately?

Here is the "eon.txt" [log file](https://gist.github.com/AlexandreBonneau/eaf396e2ba628528f005d57a9c0382bb).

Just to be sure it was not a driver problem, I launched Rocket League and everything was smooth on ultra.

**Comment by vpltd-peterm** [comment]
Nothing has changed since Sep 8, 2015

**Comment by AlexandreBonneau** [comment]
Well, that's a bummer then :\

Is the log of any help though?

**Comment by vpltd-kgalaj** [comment]
I read through the log, all seems normal :( Possibly you could profile the system performance with some free system-wide profiler like http://oprofile.sourceforge.net/ , and see where the CPUs spend their time?

**Comment by AlexandreBonneau** [comment]
Ok I checked again and all the accelerated games I tested could run very smoothly with the amdgpu driver, except Dirt Showdown.

That list include The binding of isaac afterbirth, shadow tactics, awesomenauts, Rocket league, etc.

Since I opened the bug report, I upgraded to mesa 13.0.3.
About oprofile, I'm usually not really keen to install apps that are not in the Debian repository (with the exception of Steam). Do you know about any open-source alternatives?

**Comment by AlexandreBonneau** [comment]
The problem is still seen with mesa 13.0.6-1, unfortunately.

Any other report like this on your end?

**Comment by vpltd-kgalaj** [comment]
No, no other reports :( Possibly this is a driver change bringing to light some physical issue with the graphics card on your specific machine?

**Comment by AlexandreBonneau** [comment]
I wouldn't know about the mesa driver ; I do know it worked perfectly well *before*, but not anymore.
I'm not sure if this is related, but I cannot start GRID Autosport from codemaster as well (but I could before, obviously!).

And since apparently the log is not verbose about the problem cause, does this means this issue is doomed to be stuck in the 'open' state limbo for the time being? :)

**Comment by vpltd-kgalaj** [comment]
The biggest problem with this specific issue is our inability to reproduce it. With current deadlines, we don't have the luxury of time to set it up on multiple machines and try to catch it in the act somewhere, especially as there isn't a big crowd shouting for a fix, so it must be rare and there's a chance we wouldn't be able to reproduce it even on a single machine if we tried. So, unfortunately, I can't promise a fix until we can make it happen on a machine we have somehow.

**Comment by AlexandreBonneau** [comment]
I understand, sadly.

On my end I do not see why it suddenly stopped working for this 2 specific games.

:/
